config/initializers/rack_attack.rb

```diff
@@ -1,6 +1,37 @@
 # frozen_string_literal: true
-# No more that 500 requersts per minitue per ip
-limit = ENV.fetch('HYKU_ATTACK_RATE_LIMIT', 500).to_i
+
+# No more that 20 requests per minute per ip
+limit = ENV.fetch('HYKU_ATTACK_RATE_LIMIT', 20).to_i
```

Comment:
I like how these are env vars we can update live if needed.

Reply:
Agreed, credit for these goes to @orangewolf. It's a pattern I'd love to see more often :grin: